lib/query/main.js

```diff
@@ -33,6 +33,20 @@ const execute = (conn, database, query, params) => {
   );
 };
 
+const executeInTransaction = (conn, database, transactionId, query, params) => {
+  const headers = conn.headers();
+  headers.append('Accept', mimeType(query));
+  headers.append('Content-Type', 'application/x-www-form-urlencoded');
+
+  const body = buildQuery(query, params);
+
+  return fetch(conn.uri(database, transactionId, 'query'), {
```

Comment:
You should attach the transaction back somehow. That's what I've been doing so you can string them together. Any call that requires a transaction ID, it should come back in the result.